src/components/calculator.js

```diff
@@ -29,29 +30,34 @@ const Calculator = () => {
     return 0;
   };
   return (
-    <div className="calculator-flex">
-      <div className="output">{display()}</div>
-      <div className="buttons">
-        <Button value="AC" click={handler} />
-        <Button value="+/-" click={handler} />
-        <Button value="%" click={handler} />
-        <Button value="÷" click={handler} />
-        <Button value="7" click={handler} />
-        <Button value="8" click={handler} />
-        <Button value="9" click={handler} />
-        <Button value="x" click={handler} />
-        <Button value="4" click={handler} />
-        <Button value="5" click={handler} />
-        <Button value="6" click={handler} />
-        <Button value="-" click={handler} />
-        <Button value="1" click={handler} />
-        <Button value="2" click={handler} />
-        <Button value="3" click={handler} />
-        <Button value="-" click={handler} />
-        <Button value="+" click={handler} />
-        <Button value="0" click={handler} />
-        <Button value="." click={handler} />
-        <Button value="=" click={handler} />
+    <div className="container">
+      <div className="cal-p">
+        <p>Solve some Maths</p>
+      </div>
+      <div className="calculator-flex">
+        <div className="output">{display()}</div>
+        <div className="buttons">
+          <Button value="AC" click={handler} />
+          <Button value="+/-" click={handler} />
+          <Button value="%" click={handler} />
+          <Button value="÷" click={handler} />
+          <Button value="7" click={handler} />
+          <Button value="8" click={handler} />
+          <Button value="9" click={handler} />
+          <Button value="x" click={handler} />
+          <Button value="4" click={handler} />
+          <Button value="5" click={handler} />
+          <Button value="6" click={handler} />
+          <Button value="-" click={handler} />
+          <Button value="1" click={handler} />
+          <Button value="2" click={handler} />
+          <Button value="3" click={handler} />
+          <Button value="-" click={handler} />
+          <Button value="+" click={handler} />
+          <Button value="0" click={handler} />
+          <Button value="." click={handler} />
+          <Button value="=" click={handler} />
+        </div>
       </div>
```

Comment:
> 

It works as expected though, functionality has been implemented as expected. It's just a CSS issue, if I span one of the bottom buttons it will push the others up. How is this part of today's requirements?